docs/components/ToastStackView.jsx

```diff
@@ -183,6 +192,13 @@ export default class ToastStackView extends React.PureComponent {
             description: "An additional CSS class name to apply to the container.",
             optional: true,
           },
+          {
+            name: "clearNotification",
+            type: "(id: number) => void",
+            description: "A function that clears the specified notification. Important gotcha: " +
+              "This function may be run with a timer, so make sure that the function doesn't " +
```

Comment:
I'm not sure if I understand the gotcha / how to avoid it.

Reply:
After we sync up tomorrow on the bug itself, I'd love to hear your thoughts on how to concisely guide against it here

Reply:
@mkmathur Thanks for you help coming up with a better guideline! Added in https://github.com/Clever/components/pull/350/commits/ff016cf9721e0db529e677b9342e04d0e4de400b

<img width="1022" alt="ayy" src="https://user-images.githubusercontent.com/12616928/50995307-99a6a100-14d3-11e9-8795-433b4c8ed72a.png">

Reply:
Perfect!